kitsune/wiki/forms.py

```diff
@@ -158,20 +153,34 @@ def clean_slug(self):
         return slug
 
     def clean(self):
-        c = super(DocumentForm, self).clean()
-        locale = c.get("locale")
+        cdata = super(DocumentForm, self).clean()
+        locale = cdata.get("locale")
 
         # Products are required for en-US
-        products = c.get("products")
-        if locale == settings.WIKI_DEFAULT_LANGUAGE and (not products or len(products) < 1):
+        product_ids = cdata.get("products")
+        products = Product.objects.filter(id__in=[int(product_id) for product_id in product_ids])
```

Comment:
This needs to take into account that `product_ids` can be `None`, and I think we should use `Product.active.filter(...)` here?